blog/2017-12-20-debezium-0-7-1-released.adoc

```diff
@@ -0,0 +1,33 @@
+= Debezium 0.7.1 Is Released
+jpechane
+:awestruct-tags: [ releases, mysql, mongodb, postgres, docker ]
+:awestruct-layout: blog-post
+
+Just last few days before Christmas we are releasing Debezium  *0.7.1*!
+This is only bugfix release that fixes few annoying issues that were found during first rounds of use by our community.
+All issues relate to either newly provided wal2json support or reduced risk of internal race condition improvement..
+
+https://github.com/rcoup[Robert Coup] has found a https://issues.jboss.org/browse/DBZ-512[performance regression] in situations when 0.7.0 was used with old version of Protobuf decoder.
+
+Suraj Savita (and others) has found an issue when our code failed to https://issues.jboss.org/browse/DBZ-513[crrectly detect] it runs with Amazon RDS wal2json plug-in.
+We had admitted that we are outsmarted by the JDBC driver internals and we included a distinct plugin decoder name https://issues.jboss.org/browse/DBZ-517[wal2json_rds] that bypasses detection routine and by default expects it runs against Amazon RDS instance.
```

Comment:
I don't understand "we had admitted"?

What does this change mean when using wal2json not on RDS?